legacy/helmcharts/mariadb-dbaas/templates/prebackuppod.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     /bin/sh -c "dump=$(mktemp)
     && mysqldump --max-allowed-packet=500M --events --routines --quick
     --add-locks --no-autocommit --single-transaction --no-create-db
-    --no-data --no-tablespaces
+    --no-data --no-tablespaces --skip-triggers
```

Comment:
The first command dumps the structure and the second command dumps the data. It might be better to include the triggers in the structure and not with the data? i.e. add `--skip-triggers` in the second command. Either would work I suppose.

Reply:
Sure, happy to do that :)